invenio_app_ils/records_relations/api.py

```diff
@@ -180,6 +180,8 @@ def _build_sibling(self, related_pid, relation_type):
 
         # add also title, language and edition of the sibling
         r["title"] = sibling.get("title", "")
+        r["document_type"] = sibling.get("document_type", "")
```

Comment:
I think I did something a bit silly previously here, I was adding `title` in any case, even if empty, but not language or edition if empty.
We should probably decide one way :)